src/jmdaemon/onionmc.py

```diff
@@ -641,7 +641,8 @@ def __init__(self,
         self.serverport = self.hostid
         self.tor_control_host = configdata["tor_control_host"]
         self.tor_control_port = configdata["tor_control_port"]
-        self.onion_serving_host=configdata["onion_serving_host"]
+        self.tor_control_password = configdata["tor_control_password"]

```

Comment:
```suggestion
        self.tor_control_password = configdata.get("tor_control_password")
```

afaict, this syntax would ensure `None` is returned would `tor_control_password` be absent from config